.ci/windows-steps.yaml

```diff
@@ -76,7 +76,7 @@ steps:
     msbuildVersion: $(MSBuildVersion)
     configuration: 'Release'
     msbuildArchitecture: 'x64'
-    msbuildArguments: /m /p:BuildInParallel=true
+    msbuildArguments: /m /p:BuildInParallel=false
```

Comment:
I think you have to remove the `/m` too.  Honestly I think you could remove the whole line.